### PR TITLE
Add Second Assessment Report GWP100 conversions

### PR DESCRIPTION
**Pull request by znicholls**
# Pull request

Please confirm that this pull request has done the following:

- [ ] Tests added
- [ ] Documentation added (where applicable)
- [ ] Example added (either to an existing notebook or as a new notebook, where applicable)
- [ ] Description in ``CHANGELOG.rst`` added

## Adding to CHANGELOG.rst

Please add a single line in the changelog notes similar to one of the following:

```
- (`#XX <http://link-to-pr.com>`_) Added feature which does something
- (`#XX <http://link-to-pr.com>`_) Fixed bug identified in (`#XX <http://link-to-issue.com>`_)
```

### CODE REVIEW

File: openscm/units.py [hunk 1]
```diff
@@ -217,20 +237,62 @@ def _add_gases_to_unit_registry(unit_registry, gases):
 unit_registry.define("ppb = 1000 * ppt")
 unit_registry.define("ppm = 1000 * ppb")
 
+
 # Contexts:
+context_definitions = join(dirname(__file__), "metric_values.csv")
```
Comment (swillner) [inline openscm/units.py]: Let's avoid reading files on import.

Comment (znicholls) [reply openscm/units.py]: What's a good way to store tabular data then? Only read when requested?

Comment (swillner) [reply openscm/units.py]: hmm, what about a dict for each context only including gases with values? or a dict of these mapping context name to the dict with gases->values.

Comment (swillner) [reply openscm/units.py]: though additionally lazy-loading contexts would be nice

Comment (znicholls) [reply openscm/units.py]: > what about a dict for each context only including gases with values

being able to edit these values in a csv editor would be ideal given they're almost always tables of values... worst case dicts which we then write a nice wrapper to at least look at tables is fine

**Pull request closed without merging**